netlify.toml

```diff
@@ -1,6 +1,6 @@
 [build]
   publish = "docs-production/"
-  command = "yarn boot && yarn docs:build-preview"
+  command = "yarn bootstrap && yarn docs:build-preview"
 
   [build.environment]
     YARN_VERSION = "1.3.2"
```

Comment:
Maybe also update your Yarn version everywhere, good fixes I am sure for workspaces

Reply:
I'll do it in another PR.